frontend/configureServer.sh

```diff
@@ -16,21 +16,28 @@ storage_service_host=localhost
 storage_service_port=8090
 
 # Check if the required env variables are set otherwise localhost will be used.
+## TODO: Remove these lines when Pipeline is configured with new API env vars
 if [ -n "$PROD_API_HOST" ]; then
-    prod_host=$PROD_API_HOST
+    api_host=$PROD_API_HOST
 fi
-
 if [ -n "$PROD_API_PORT" ]; then
-    prod_port=$PROD_API_PORT
+    api_port=$PROD_API_PORT
 fi
-
 if [ -n "$TEST_API_HOST" ]; then
-    test_host=$TEST_API_HOST
+    api_host=$TEST_API_HOST
 fi
-
 if [ -n "$TEST_API_PORT" ]; then
-    test_port=$TEST_API_PORT
+    api_port=$TEST_API_PORT
+fi
+### TODO end
+if [ -n "$API_HOST" ]; then
+    api_host=$API_HOST
```

Comment:
ToDo: This env vars must be change in the pipeline

Reply:
resolve TODO